Us398/typography find help

leahtbriggs opened the pull request.

leahtbriggs committed:
Storyless: Merge from integration

leahtbriggs committed:
US401T416: Make sure all app colors are using the colors.js page

Make sure all app colors are using the colors.js page, changed the pchptsdaccordian.js page to use colors.js page.

leahtbriggs committed:
Storyless: Test changes

Test changes associated with color changes.

leahtbriggs committed:
Merge branch 'integration' of https://github.com/gcarvajal97/pch_ptsd into US398/TypographyFindHelp

leahtbriggs committed:
US398T409: Conform typography of quick tips screen

Conform typography of quick tips screen to fit the PCH branding standards.

leahtbriggs committed:
US398T410: Conform typography of Deal with Worries screen

Conform typography of Deal with Worries screen and resolved errors on the Quick Tips screen.

leahtbriggs committed:
US398T411: Conform typography of How to Talk screen

Conform typography of How to Talk screen.

leahtbriggs committed:
US398T413: Conform typography of Outside Help screen and subscreens

Conform typography of Outside Help screen and subscreens to fit with the PCH typography standards.

leahtbriggs committed:
US398T412: Conform typography of Self Care screen and subscreens

Conform typography of Self Care screen and subscreens to PCH typography standards.

leahtbriggs committed:
US398T414: Conform typography of Injury Pain Care screen and subscreens

Conform typography of Injury Pain Care screen and subscreens to fit PCH standards.

leahtbriggs committed:
US398T415: Conform typography of Healthcare Team screen and subscreens

Conform typography of Healthcare Team screen and subscreens to PCH standards.

leahtbriggs committed:
Merge branch 'integration' of https://github.com/gcarvajal97/pch_ptsd into US398/TypographyFindHelp

leahtbriggs committed:
Storyless: Update snapshots

gcarvajal97 commented on findHelpScreens/DealWithWorries.js:
```diff
@@ -187,6 +188,7 @@ const styles = StyleSheet.create({
         marginHorizontal: 18,
         textAlign: "left",
         alignSelf: "stretch",
+        fontFamily: "avenir-medium",
```
I thought avenir-medium was already set to be the default font?

tullosjr commented on findHelpScreens/HealthcareTeamScreens/AfterTheHospital.js:
```diff
@@ -52,12 +52,13 @@ const styles = StyleSheet.create({
   paragraph: {
     textAlign: 'left',
     fontSize: 14,
+    fontFamily: 'avenir-medium',
```
This is unnecessary

tullosjr commented on findHelpScreens/HealthcareTeamScreens/HealthcareTeam.js:
```diff
@@ -111,7 +111,7 @@ export default class HealthcareTeam extends Component {
                     <Card
                         image={healthTeam}
-                        featuredTitleStyle={{ fontSize: 40 }}
+                        featuredTitleStyle={{ fontSize: 40, fontFamily: 'avenir-medium'}}
```
The title font isn't overridden on Android:
![image](https://user-images.githubusercontent.com/22266870/79055159-5ffda200-7c10-11ea-98f0-30dbcf0722da.png)

tullosjr replied:
We're also missing fonts on the accordion:
![image](https://user-images.githubusercontent.com/22266870/79055341-e8c90d80-7c11-11ea-96bf-99993a2e98ed.png)

tullosjr commented on findHelpScreens/HealthcareTeamScreens/HealthcareTeam.js:
```diff
@@ -183,10 +183,12 @@ const styles = StyleSheet.create({
     },
     headerText: {
         fontSize: 16,
+        fontFamily: 'avenir-medium',
```
This should be 'avenir-heavy' with no fontWeight

tullosjr commented on findHelpScreens/HealthcareTeamScreens/InTheED.js:
```diff
@@ -43,6 +43,7 @@ const styles = StyleSheet.create({
   paragraph: {
     textAlign:'left',
     fontSize: 14,
+    fontFamily: 'avenir-medium',
```
This is unnecessary.

tullosjr commented on findHelpScreens/HealthcareTeamScreens/InTheHospital.js:
```diff
@@ -71,13 +71,14 @@ const styles = StyleSheet.create({
   },
   paragraph: {
     textAlign:'left',
+    fontFamily: 'avenir-medium',
```
This is unnecessary.

tullosjr commented on findHelpScreens/HowToTalk.js:
```diff
@@ -121,11 +120,12 @@ const styles = StyleSheet.create({
         paddingLeft: 15,
         paddingRight:15,
         fontSize: 15,
+        fontFamily: "avenir-medium",
```
This is unnecessary.

tullosjr commented on findHelpScreens/HowToTalk.js:
```diff
@@ -139,14 +139,15 @@ const styles = StyleSheet.create({
       marginHorizontal:30,
       textAlign:'left',
       alignSelf:'stretch',
+      fontFamily: "avenir-medium",
```
This is unnecessary.

tullosjr commented on findHelpScreens/HowToTalk.js:
```diff
@@ -155,5 +156,6 @@ const styles = StyleSheet.create({
         paddingHorizontal:5,
         fontSize: 14,
         marginTop: 5,
+        fontFamily: "avenir-medium"
```
This is unnecessary.

tullosjr commented on findHelpScreens/InjuryPainCareScreens/AdbominalTraumaScreen.js:
```diff
@@ -44,12 +44,13 @@ const styles = StyleSheet.create({
   paragraph: {
     textAlign: 'left',
     fontSize: 14,
+    fontFamily: "avenir-medium",
```
This is unnecessary.

tullosjr commented on findHelpScreens/InjuryPainCareScreens/HeadInjuryScreen.js:
```diff
@@ -40,17 +40,19 @@ const styles = StyleSheet.create({
   paragraph: {
     textAlign: 'left',
     fontSize: 14,
+    fontFamily: 'avenir-medium',
```
This is unnecessary.

tullosjr commented on findHelpScreens/InjuryPainCareScreens/HeadInjuryScreen.js:
```diff
@@ -40,17 +40,19 @@ const styles = StyleSheet.create({
   },
   link: {
     fontSize: 14,
+    fontFamily: 'avenir-medium',
```
This is unnecessary.

tullosjr commented on findHelpScreens/HealthcareTeamScreens/HealthcareTeam.js:
```diff
@@ -213,10 +215,11 @@ const styles = StyleSheet.create({
     },
     selectTitle: {
         fontSize: 14,
+        fontFamily: 'avenir-medium',
```
This should be 'avenir-heavy' with no fontWeight property.

tullosjr commented on findHelpScreens/InjuryPainCareScreens/InjuryPainCare.js:
```diff
@@ -209,12 +209,14 @@ const styles = StyleSheet.create({
         fontSize: 16,
-        fontWeight: "500"
+        fontWeight: "500",
+        fontFamily: 'avenir-medium',
```
This should be 'avenir-heavy' with no fontWeight

tullosjr commented on findHelpScreens/InjuryPainCareScreens/InjuryPainCare.js:
```diff
@@ -239,10 +241,11 @@ const styles = StyleSheet.create({
     },
     selectTitle: {
         fontSize: 14,
+        fontFamily: 'avenir-medium',
```
This should be 'avenir-heavy' with no fontWeight

tullosjr commented on findHelpScreens/InjuryPainCareScreens/InjuryPainCare.js:
```diff
@@ -253,11 +256,12 @@ const styles = StyleSheet.create({
         paddingLeft: 15,
         paddingRight: 15,
         fontSize: 15,
+        fontFamily: 'avenir-medium',
```
This 'subTitle' style is unused.

tullosjr commented on findHelpScreens/InjuryPainCareScreens/InjuryPreventionTipsScreen.js:
```diff
@@ -36,19 +36,21 @@ const styles = StyleSheet.create({
   },
   paragraph: {
     textAlign: 'left',
+    fontFamily: 'avenir-medium',
```
This is unnecessary.

tullosjr commented on findHelpScreens/InjuryPainCareScreens/InjuryPainCare.js:
```diff
@@ -131,7 +131,7 @@ export default class InjuryPainCare extends Component {
                 <ScrollView contentContainerStyle={{ paddingTop: 0 }}>
```
Card and accordion fonts aren't applied on Android:
![image](https://user-images.githubusercontent.com/22266870/79055462-f763f480-7c12-11ea-9229-7a7e85472426.png)

tullosjr commented on findHelpScreens/InjuryPainCareScreens/InjuryPreventionTipsScreen.js:
```diff
@@ -36,19 +36,21 @@ const styles = StyleSheet.create({
   link: {
     fontSize: 14,
     color: colors.accent,
+    fontFamily: 'avenir-medium',
```
This is unnecessary.

tullosjr commented on findHelpScreens/InjuryPainCareScreens/ManagingPainScreen.js:
```diff
@@ -125,18 +125,20 @@ const styles = StyleSheet.create({
     paragraph: {
       textAlign: 'left',
       fontSize: 14,
+      fontFamily: 'avenir-medium',
```
This is unnecessary.

tullosjr commented on findHelpScreens/InjuryPainCareScreens/ManagingPainScreen.js:
```diff
@@ -125,18 +125,20 @@ const styles = StyleSheet.create({
       fontSize: 14,
       marginVertical: 2,
-      textDecorationLine: 'underline',
+      fontFamily: 'avenir-medium',
```
This is unnecessary.

tullosjr commented on findHelpScreens/InjuryPainCareScreens/TakingCareofCastScreen.js:
```diff
@@ -73,12 +73,13 @@ const styles = StyleSheet.create({
   paragraph: {
     textAlign: 'left',
     fontSize: 14,
+    fontFamily: 'avenir-medium',
```
This is unnecessary.

tullosjr commented on findHelpScreens/InjuryPainCareScreens/UsingCrutchesScreen.js:
```diff
@@ -65,12 +65,13 @@ const styles = StyleSheet.create({
   paragraph: {
     textAlign: 'left',
     fontSize: 14,
+    fontFamily: "avenir-medium",
```
This is unnecessary.

tullosjr commented on findHelpScreens/QuickTipsScreens/QuickTips.js:
```diff
@@ -79,12 +79,14 @@ const styles = StyleSheet.create({
       color: colors.primary.white, 
```
Looks like every card we have in Find Help is like this:
![image](https://user-images.githubusercontent.com/22266870/79055549-87a23980-7c13-11ea-99af-5550ac4832ad.png)

tullosjr commented on findHelpScreens/QuickTipsScreens/QuickTips.js:
```diff
@@ -107,6 +110,7 @@ const styles = StyleSheet.create({
       paddingVertical:1, 
       paddingHorizontal:5,
       fontSize: 14,
+      fontFamily: "avenir-medium",
```
This is unnecessary.

tullosjr commented on findHelpScreens/outsideHelpScreens/HowToLook.js:
```diff
@@ -43,18 +43,20 @@ const styles = StyleSheet.create({
   paragraph: {
     textAlign: 'left',
     fontSize: 14,
+    fontFamily: "avenir-medium",
```
This is unnecessary.

tullosjr commented on findHelpScreens/outsideHelpScreens/HowToLook.js:
```diff
@@ -43,18 +43,20 @@ const styles = StyleSheet.create({
   link: {
     fontSize: 14,
     color: colors.accent,
+    fontFamily: "avenir-medium",
```
This is unnecessary.

tullosjr commented on findHelpScreens/outsideHelpScreens/WhenAndHowToLookForYourself.js:
```diff
@@ -39,17 +39,19 @@ const styles = StyleSheet.create({
     textAlign: 'left',
     fontSize: 14,
     marginVertical: 2,
+    fontFamily: "avenir-medium"
```
This is unnecessary.

tullosjr commented on findHelpScreens/outsideHelpScreens/WhenAndHowToLookForYourself.js:
```diff
@@ -39,17 +39,19 @@ const styles = StyleSheet.create({
     fontSize: 14,
     color: colors.accent,
     textDecorationLine: 'underline',
+    fontFamily: "avenir-medium",
```
This is unnecessary.

tullosjr commented on findHelpScreens/outsideHelpScreens/WhenToGetOutsideHelp.js:
```diff
@@ -105,7 +105,7 @@ export default class WhenToGetOutsideHelpScreen extends Component {
                 <ScrollView contentContainerStyle={{ paddingTop: 0 }}>
                     <Card
```
![image](https://user-images.githubusercontent.com/22266870/79055645-4cecd100-7c14-11ea-911a-c4e42535fd37.png)

tullosjr commented on findHelpScreens/outsideHelpScreens/WhenToGetOutsideHelp.js:
```diff
@@ -177,15 +177,17 @@ const styles = StyleSheet.create({
         fontSize: 16,
-        fontWeight: "500"
+        fontWeight: "500",
+        fontFamily: "avenir-medium",
```
Should be 'avenir-heavy' with no fontWeight.

tullosjr commented on findHelpScreens/outsideHelpScreens/WhenToGetOutsideHelp.js:
```diff
@@ -177,15 +177,17 @@ const styles = StyleSheet.create({
         marginTop: 0,
-        marginBottom: 4
+        marginBottom: 4,
+        fontFamily: "avenir-medium",
```
This is unnecessary.

tullosjr commented on findHelpScreens/outsideHelpScreens/WhenToGetOutsideHelp.js:
```diff
@@ -207,11 +209,12 @@ const styles = StyleSheet.create({
     selectTitle: {
         fontSize: 14,
         fontWeight: "500",
+        fontFamily: "avenir-medium",
```
This should be 'avenir-heavy' with no font weight.

tullosjr commented on findHelpScreens/outsideHelpScreens/WhenToLook.js:
```diff
@@ -40,18 +40,20 @@ const styles = StyleSheet.create({
   paragraph: {
     textAlign: 'left',
     fontSize: 14,
+    fontFamily: "avenir-medium",
```
This is unnecessary.

tullosjr commented on findHelpScreens/outsideHelpScreens/WhenToLook.js:
```diff
@@ -40,18 +40,20 @@ const styles = StyleSheet.create({
   link: {
     fontSize: 14,
     color: colors.accent,
+    fontFamily: "avenir-medium",
```
This is unnecessary.

tullosjr commented on findHelpScreens/selfCareScreens/AnxiousJumpy.js:
```diff
@@ -58,12 +58,13 @@ const styles = StyleSheet.create({
     paragraph: {
       textAlign: 'left',
       fontSize: 14,
+      fontFamily: "avenir-medium",
```
This is unnecessary.

tullosjr commented on findHelpScreens/selfCareScreens/Avoidance.js:
```diff
@@ -47,13 +47,14 @@ const styles = StyleSheet.create({
     },
     paragraph: {
       textAlign: 'left',
+      fontFamily: "avenir-medium",
```
This is unnecessary.

tullosjr commented on findHelpScreens/selfCareScreens/RelivingWhat.js:
```diff
@@ -42,13 +42,14 @@ const styles = StyleSheet.create({
     },
     paragraph: {
       textAlign: 'left',
+      fontFamily: "avenir-medium",
```
This is unnecessary.

tullosjr commented on findHelpScreens/selfCareScreens/SelfCare.js:
```diff
@@ -192,13 +192,15 @@ const styles = StyleSheet.create({
         fontSize: 16,
-        fontWeight: "500"
+        fontWeight: "500",
+        fontFamily: "avenir-medium"
```
This should be 'avenir-heavy' with no fontWeight.

tullosjr commented on findHelpScreens/selfCareScreens/SelfCare.js:
```diff
@@ -192,13 +192,15 @@ const styles = StyleSheet.create({
         paddingVertical: 0,
         paddingHorizontal: 4,
         fontSize: 14,
+        fontFamily: "avenir-medium",
```
This is unnecessary.

tullosjr commented on findHelpScreens/selfCareScreens/SelfCare.js:
```diff
@@ -222,11 +224,12 @@ const styles = StyleSheet.create({
     selectTitle: {
         fontSize: 14,
         fontWeight: "500",
+        fontFamily: "avenir-medium",
```
This should be 'avenir-heavy' with no fontWeight.

tullosjr commented on findHelpScreens/selfCareScreens/SelfCare.js:
```diff
@@ -236,11 +239,12 @@ const styles = StyleSheet.create({
         paddingLeft: 15,
         paddingRight: 15,
         fontSize: 15,
+        fontFamily: "avenir-medium",
```
This is unnecessary.

tullosjr commented on findHelpScreens/selfCareScreens/SelfCareCard.js:
```diff
@@ -12,7 +12,7 @@ export default class SelfCareCard extends Component {
         return(
         <Card
```
![image](https://user-images.githubusercontent.com/22266870/79055730-ea480500-7c14-11ea-871d-cafe724eebb6.png)

tullosjr commented on findHelpScreens/selfCareScreens/SelfCareParents.js:
```diff
@@ -45,12 +45,13 @@ const styles = StyleSheet.create({
     paragraph: {
       textAlign: 'left',
       fontSize: 14,
+      fontFamily: "avenir-medium",
```
This is unnecessary.

leahtbriggs committed:
Storyless: Updated snap tests

leahtbriggs committed:
Storyless: Resolved comments and updated snapshots

Resolved suggestions from team mates and updated snapshots

leahtbriggs committed:
Storyless: Forgot one change, updated snapshots

Forgot one change in my previous commits and, updated snapshots to reflect that

leahtbriggs committed:
Merge branch 'integration' of https://github.com/gcarvajal97/pch_ptsd into US398/TypographyFindHelp

leahtbriggs committed:
Storyless: Updated name in file

Updated name in file

Commit:
Fix quick tips card font

Commit:
Fix ListItem font on find help list

Commit:
Fix numbered section font in quick tips

Commit:
Add bold font

Commit:
Fix injury and pain card title font on android

leahtbriggs committed:
Fix self care card font

leahtbriggs committed:
Fix outside help card font

leahtbriggs committed:
Merge branch 'US398/TypographyFindHelp' of https://github.com/gcarvajal97/pch_ptsd into US398/TypographyFindHelp

leahtbriggs committed:
Update Tests

Commit:
Update healthcare card font to bold
